code/game/objects/effects/decals/turfdecal/markings.dm

```diff
@@ -246,6 +246,19 @@
 /obj/effect/turf_decal/siding/red/end
 	icon_state = "siding_end"
 
+// Skyrat Edit: Start - Adds red variant that doesnt break 'bluesec' but allows mappers to use red elsewhere
```

Comment:
modularize this

Reply:
yeah was hoping to keep them in the same group